sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -713,7 +713,7 @@
                         "maxWords": "",
                         "pattern": "",
                         "customMessage": "",
-                        "custom": "valid = /^(11|09|52|01|03|05|16|23|24|25|42|27|30|38|40|45|50|54|55|13|28|31|44|60|04|10|14|15|29|41|26|22|43|51|39|12|21|46|47|48){1}\\.[a-zA-Z]{4}$/.test(input) ? true : 'Incorrect Format';",
+                        "custom": "valid = /^(11|09|52|01|03|05|16|23|24|25|42|27|30|38|40|45|50|54|55|13|28|31|44|60|04|10|14|15|29|41|26|22|43|51|39|12|21|46|47|48){1}\\.[0-9]{4}$/.test(input) ? true : 'Incorrect Format';",
```

Comment:
I missed that in the previous PR. Is it not possible to avoid the custom code and just add the regex to the below field in the validation section?
![image](https://user-images.githubusercontent.com/61259237/185202034-16b2bffe-d59c-4b57-855e-01af2f905065.png)

And the error message can be configured here also if I am not wrong.
![image](https://user-images.githubusercontent.com/61259237/185202034-16b2bffe-d59c-4b57-855e-01af2f905065.png)

Reply:
Fixed.